deps/ncrypto/ncrypto.cc

```diff
@@ -4190,6 +4190,36 @@ DataPointer hashDigest(const Buffer<const unsigned char>& buf,
   return data.resize(result_size);
 }
 
+DataPointer xofHashDigest(const Buffer<const unsigned char>& buf,
```

Comment:
@jasnell @anonrig Do we currently need to upstream changes to deps/ncrypto? I see that https://github.com/nodejs/ncrypto/issues/7 is still open and it seems the source of truth is still in deps/ncrypto (i.e. it is more advanced than https://github.com/nodejs/ncrypto)

Reply:
Put updates into `deps/ncrypto` for now. I'm going to be rebasing the repo in the coming couple of weeks and will hopefully soon switch over.